src/service/gnuradio/examples/function_generator.grc

```diff
@@ -15,18 +15,20 @@ blocks:
         - 300000000   # 300 ms in ns
         - 350000000   # 350 ms in ns
         - 550000000   # 550 ms in ns
+        - 560000000   # 560 ms in ns
         - 650000000   # 650 ms in ns
         - 800000000   # 800 ms in ns
         - 850000000   # 850 ms in ns
       tag_values:
-        - "CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=1"
-        - "CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=2"
-        - "CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=3"
-        - "CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=4"
-        - "CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=5"
-        - "CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=6"
-        - "CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=7"
-        - "CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=8"
+        - "CMD_BP_START/FAIR.SELECTOR.C=1:S=1:P=1"
```

Comment:
This is how I could make the matching work, with e.g. `[CMD_BP_START/FAIR.SELECTOR.C=1:S=1:P=5, CMD_BP_START/FAIR.SELECTOR.C=1:S=1:P=8]` (multiplexed) or `CMD_DIAG_TRIGGER1 (triggered) as filter setting setting.

With the original file, I get tags where both TRIGGER_NAME and CONTEXT are e.g. `CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=2` (i.e. both contain the full string), which then doesn't match `[CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=5, CMD_BP_START:FAIR.SELECTOR.C=1:S=1:P=8]`. @drslebedev can we look at this together?

Reply:
Thank you for finding that, @frankosterfeld. Your changes are correct—it was a bug; we should use “/”.

Reply:
Great :)